packages/react-native-reanimated/Common/cpp/reanimated/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -348,7 +345,7 @@ jsi::Value NativeReanimatedModule::getViewProp(
   const auto funPtr = std::make_shared<jsi::Function>(
       callback.getObject(rnRuntime).asFunction(rnRuntime));
   const auto shadowNode = shadowNodeFromValue(rnRuntime, shadowNodeWrapper);
-  uiScheduler_->scheduleOnUI([=]() {
+  uiScheduler_->scheduleOnUI([=, this]() {
```

Comment:
Why do we need this?

Reply:
I should revert this. Implicit capture of `this` is deprecated in NDK27. I fixed it some time ago, but it turns out syntax `[=, this]` is not available in NDK23. Generally this is a pain in the bum.

PRs and issues for reference:

- #6495
- #6466
- #6512

Reply:
I will address those capture issues in a separate PR.